packages/html/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@prairielearn/html",
   "version": "3.0.1",
   "main": "dist/index.js",
+  "sideEffects": false,
```

Comment:
This is a convention established by webpack a long time ago; esbuild also understands this convention. This instructs bundlers that importing/requiring this package doesn't have any side effects and thus that it can be safely removed from the bundle if nothing actually uses it. See https://webpack.js.org/guides/tree-shaking/ for more details.